controllers/keyboard.py

```diff
@@ -81,9 +83,16 @@ def start_processing_input(system_queue, demo_input_queue):
                 elif event.key == K_SPACE:
                     demo_input_queue.put("PRI_R")
                 elif event.key == K_n:
-                    demo_input_queue.put("SEC_R")
-                elif event.key == K_RCTRL:
-                    demo_input_queue.put("SEL_R")
+                    system_queue.put("SEC_R")
+
+                # Note: this has purposely been removed. I argue that we don't
```

Comment:
I agree with this sentiment. I reckon the comment should be removed if it does get merged with `dev`

Reply:
I was on the fence but was thinking of keeping it in so that someone doesn't try to add it later. I will remove it but rename `SEL_P` to `SEL` so that it makes it more clear that we don't care about releasing (`_R`) versus pressing (`_P`).